### PR TITLE
fix type mapping for n6 parser

### PR DESCRIPTION
**Pull request by wagner-certat**
fixes certtools/intelmq#738

mappings for idea are confirmed by @pharook

### CODE REVIEW

File: intelmq/bots/parsers/n6/parser_n6stomp.py [hunk 1]
```diff
@@ -59,8 +59,8 @@
                            "type": "malware", "identifier": "malware"}
 mapping['vulnerable']   = {"taxonomy": "Vulnerable",
                            "type": "other", "identifier": "vulnerable"}
-mapping['other']        = {"taxonomy": "Vulnerable",
-                           "type": "unknown", "identifier": "unknown"}
+mapping['other']        = {"taxonomy": "Other",
```
Comment (aaronkaplan) [inline intelmq/bots/parsers/n6/parser_n6stomp.py]: agreed

File: intelmq/bots/parsers/n6/parser_n6stomp.py [hunk 2]
```diff
@@ -41,10 +41,10 @@
                              "identifier": "malware configuration"}
 mapping['phish']        = {"taxonomy": "Fraud",
                            "type": "phishing", "identifier": "phishing"}
-mapping['proxy']        = {"taxonomy": "Vulnerable",
-                           "type": "proxy", "identifier": "open proxy"}
-mapping['sandbox-url']  = {"taxonomy": "ignore",
-                           "type": "ignore", "identifier": "ignore me"}
+mapping['proxy']        = {"taxonomy": "Other",
```
Comment (aaronkaplan) [inline intelmq/bots/parsers/n6/parser_n6stomp.py]: might be replaced in the future by the taxonomy "accessible". But that's the future....

File: intelmq/bots/parsers/n6/parser_n6stomp.py [hunk 3]
```diff
@@ -41,10 +41,10 @@
...
+                           "type": "proxy", "identifier": "openproxy"}
+mapping['sandbox-url']  = {"taxonomy": "malicious code",
```
Comment (aaronkaplan) [inline intelmq/bots/parsers/n6/parser_n6stomp.py]: ACK

**Comment by aaronkaplan** [comment]
@wagner-certat I think we can merge this.

**Comment by wagner-certat** [comment]
Thanks for the review @aaronkaplan 

I additionally added details to the NEWS file including an SQL statement.

**Pull request merged**